components/service/glean/src/main/java/mozilla/components/service/glean/scheduler/GleanLifecycleObserver.kt

```diff
@@ -24,6 +24,10 @@ internal class GleanLifecycleObserver : LifecycleObserver {
         // on foreground.
         Baseline.duration.stopAndSum()
         Glean.handleEvent(Glean.PingEvent.Background)
+
+        // This will check the metrics ping schedule to determine whether it's time to send a new
```

Comment:
Let's use the comment as a way to explain why we're not simply adding this to `Glean.PingEvent.Background`